_includes/introduction-weave-cloud-and-the-demo-app.md

```diff
@@ -0,0 +1,8 @@
+### Introduction: Weave Cloud and the Demo App
+
+In this guide you will learn how Weave Cloud can help you to understand a microservices app. You will deploy
```

Comment:
The heading and intro text here should include something about Docker for Mac, and probably doesn't belong in an include.